recipes-nodes/config-parser/render-confg.bb

```diff
@@ -0,0 +1,46 @@
+SUMMARY = "Mustache templates renderer wrapper"
+DESCRIPTION = "Render a mustache template using the provided JSON input configuration"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+FILESEXTRAPATHS:prepend := "${THISDIR}/:"
+SRC_URI = file://render-config.sh"
+
+S = "${WORKDIR}"
+
+python () {
+    # Check if INIT_CONFIG_URL is set in the environment or in local.conf
+    init_config_url = d.getVar('INIT_CONFIG_URL')
+
+    if init_config_url is None:
+        # If not set, check the original environment
+        origenv = d.getVar("BB_ORIGENV", False)
+        if origenv:
+            init_config_url = origenv.getVar('INIT_CONFIG_URL')
+
+    if init_config_url:
+        # If INIT_CONFIG_URL is set (to any non-empty value), keep its value
+        d.setVar('INIT_CONFIG_URL', init_config_url)
+}
+
+do_install() {
+    # Create necessary directories
+    install -d ${D}${bindir}
+    install -d ${D}${sysconfdir}
+    install -d ${D}${sysconfdir}/init.d
+    install -d ${D}${sysconfdir}/profile.d
```

Comment:
I guess, this is not necessary anymore since you are not exporting any env variables to all users anymore